spec/api.json

```diff
@@ -971,7 +971,8 @@
             "type": "string"
           },
           "schema": {
-            "$ref": "#/components/schemas/identitySchema"
```

Comment:
Why this change?

Reply:
This was generated by `make sdk`. This is a result of using type alias instead of type definition